src/shots/shots.ts

```diff
@@ -15,9 +15,10 @@ const takeScreenShot = async ({
   browser: Browser;
   shotItem: ShotItem;
   logger: (message: string, ...rest: unknown[]) => void;
-}) => {
+}): Promise<boolean> => {
   const context = await browser.newContext(shotItem.browserConfig);
   const page = await context.newPage();
+  let success = false;
```

Comment:
Why do we need this?

Reply:
A bit lower it is set to `true` if the screenshot could be taken successfully.
I introduced this so we can display a different message if the shot didn't happen.